docs/content/guides/rows/rows-sorting.md

```diff
@@ -0,0 +1,3432 @@
+
+:::
+
+::: only-for react
+
+```jsx
+<HotTable
+  columnSorting={{
+    // at initialization, sort data by the first column, in ascending order
+    initialConfig: {
+      column: 0,
+      sortOrder: 'asc',  // for descending order, use `'desc'`
+    },
+  }}
+/>
+```
+
+:::
+
+## Add custom sort icons
+
+The default sort icons (↑↓) are encoded in Base64. You can replace them with PNG files by changing the `background-image`
+property in the following pseudo-elements of Handsontable's CSS:
+- `.columnSorting.ascending::before`
+- `.columnSorting.descending::before`
+
+::: only-for javascript
+
+::: example #example6 --html 1 --js 2 --css 3
+
+```html
+<div id="example6"></div>
+```
+
+```js
+import Handsontable from 'handsontable';
+import 'handsontable/dist/handsontable.full.min.css';
+
+const container = document.querySelector('#example6');
+
+const handsontableInstance = new Handsontable(container, {
+  data: [
+    {
+      brand: 'Jetpulse',
+      model: 'Racing Socks',
+      price: 30,
+      sellDate: '11/10/2023',
+      sellTime: '01:23',
+      inStock: false,
+    },
+    {
+      brand: 'Gigabox',
+      model: 'HL Mountain Frame',
+      price: 1890.90,
+      sellDate: '03/05/2023',
+      sellTime: '11:27',
+      inStock: false,
+    },
+    {
+      brand: 'Camido',
+      model: 'Cycling Cap',
+      price: 130.10,
+      sellDate: '27/03/2023',
+      sellTime: '03:17',
+      inStock: true,
+    },
+    {
+      brand: 'Chatterpoint',
+      model: 'Road Tire Tube',
+      price: 59,
+      sellDate: '28/08/2023',
+      sellTime: '08:01',
+      inStock: true,
+    },
+    {
+      brand: 'Eidel',
+      model: 'HL Road Tire',
+      price: 279.99,
+      sellDate: '02/10/2023',
+      sellTime: '13:23',
+      inStock: true,
+    },
+  ],
+  columns: [
+    {
+      title: 'Brand',
+      type: 'text',
+      data: 'brand',
+    },
+    {
+      title: 'Model',
+      type: 'text',
+      data: 'model',
+    },
+    {
+      title: 'Price',
+      type: 'numeric',
+      data: 'price',
+      numericFormat: {
+        pattern: '$ 0,0.00',
+        culture: 'en-US'
+      },
+      className: 'htLeft',
+    },
+    {
+      title: 'Date',
+      type: 'date',
+      data: 'sellDate',
+      className: 'htRight',
+    },
+    {
+      title: 'Time',
+      type: 'time',
+      data: 'sellTime',
+      className: 'htRight',
+    },
+    {
+      title: 'In stock',
+      type: 'checkbox',
+      data: 'inStock',
+      className: 'htCenter',
+    },
+  ],
+  className: 'custom-sort-icon-1',
+  columnSorting: {
+    initialConfig: {
+      column: 1,
+      sortOrder: 'desc',
+    },
+  },
+  licenseKey: 'non-commercial-and-evaluation',
+});
+```
+
+```css
+/* the icon for both ascending and descending order */
+.custom-sort-icon-1 span.colHeader.columnSorting.ascending::before,
+.custom-sort-icon-1 span.colHeader.columnSorting.descending::before {
+  background-image: url("../../../img/custom_sort_icon.png") !important;
```

Comment:
```suggestion
  background-image: url("../../../img/custom_sort_icon.png") !important;
```
This demo does not work in jsfiddle or after copying/pasting the snippet to the developer's app. I'd suggest converting the icon file to base64 to get rid of mentioned problems.